src/components/menu/menu-full-screen/menu-full-screen.spec.tsx

```diff
@@ -282,6 +283,24 @@ describe("MenuFullscreen", () => {
         wrapper.find(StyledMenuFullscreen)
       );
     });
+
+    testStyledSystemPadding(
+      (props) => (
+        <MenuContext.Provider
+          value={{
+            menuType: "light",
+            openSubmenuId: null,
+            inFullscreenView: true,
+            inMenu: true,
+            setOpenSubmenuId: () => {},
+          }}
+        >
+          <MenuItem {...props}>Foo</MenuItem>
+        </MenuContext.Provider>
+      ),
+      { pt: "16px", pb: "16px" },
```

Comment:
**comment** (non-blocking): I'm not sure I like testing passing in padding values that are the same as the defaults. I know they're not the same as strings since you changed the defaults to the token values, so it's far from a useless test, but I'd still personally feel happier if you tested different values, like I dunno, 10px (or anything but 16 really).

But not a big deal since the test should still fail if the padding props are broken on MenuItem.